lib/looker-sdk/default.rb

```diff
@@ -31,7 +31,7 @@ module LookerSDK
   module Default
 
     # Default API endpoint look TODO update this as needed
-    API_ENDPOINT = "https://localhost:19999/api/3.0/".freeze
+    API_ENDPOINT = "https://localhost:19999/api/4.0/".freeze
```

Comment:
Had to do this to get tests passing in my local env

Reply:
That's a change we definitely should have had earlier since 3.0 is dead and 4.0 is our standard API now